package.json

```diff
@@ -83,6 +84,7 @@
     "flow-status-webpack-plugin": "^0.1.4",
     "glob": "^5.0.15",
     "html-webpack-plugin": "^2.14.0",
+    "insightful": "^1.0.0",
```

Comment:
We should remember to get rid of this before merging. No need to install my fake data library when building Metabase :P